### DISCUSSION TITLE
what is the best place to see list of all available plugins in nixvim?

### DISCUSSION THREAD

**Discussion by mairs8** [nix-community/nixvim#2311] [Q&A] [answered]
I want to check if nixvim has a neovim plugin available within it. I can't find anything that references such a list in the readme.

**Comment by GaetanLepage** [comment, marked as answer]
The best spot is definitely the [documentation](https://nix-community.github.io/nixvim/) (in the side bar, you can find a "plugins" menu).

We also have a fast option search engine: https://nix-community.github.io/nixvim/search/

**Reply by GaetanLepage** [reply]
If you use the stable branch of nixvim, see the corresponding documentation: https://nix-community.github.io/nixvim/stable/

**Reply by mairs8** [reply]
thank you. doesn't have the 'LazyDev' plugin i was after. This one is available in the LazyVim Distro. How is adding to plugins to Nixvim prioritised. I believe it would be good to have the plugins pre-installed in existing distros to be prioritised in some way as many people will be looking to switch from LazyVim, NVChad, LunarVim, AstroVim etc. and would like feature parity. Thanks again for the great plugin.

**Reply by MattSturgeon** [reply]
> How is adding to plugins to Nixvim prioritised

People add what they're interested in, or sometimes they'll add what seems a popular request.

We do our best to make contributing as seamless as possible (see our [CONTRIBUTING](https://github.com/nix-community/nixvim/blob/main/CONTRIBUTING.md) file), and we're always looking for feedback on how to lower the bar for new contributions!
